markdown/4.x/en/ff-suggest.md

```diff
@@ -199,3 +199,18 @@ See the following example:
     });
 </script>
 ```
+
+## On-Click redirect
+The expected behavior after clicking on `ff-suggest-item` element is a redirection to the clicked record's page.
+`ff-suggest` support this functionality if only `deeplink` field is available in clicked item attributes data.
+The `deeplink` field is populated using the data from data feed export column with a field role with the same name.
+
+**Note**: `ff-suggest` is redirecting both absolute and relative URLs.
+
+If no `deeplink` is provided for anitem which is of `productName` type then `ff-suggest` will try to obtain its value by emitting additional request.
+Depending on used FACT-Finder version it would be a search - for versions lower than `7.3`, or record API call - for newer versions.
+Then`ff-suggest` will try to redirect user once more, this time using the data received from the response to the mentioned request.
+
+If clicked item is not of `productName` type, a standard search request will be issued to FACT-Finder with additional parameter `queryFromSuggest` which is unique to `ff-suggest` and might be used to identify search request issued by this element.
+
+**Note**: Setting `ffPreventDefault` on a given item before clicking it will prevent the redirection.
```

Comment:
The value `ffPreventDefault` is set to is important I think. We should mention it explicitly.
A tiny example would make it even easier for users I think.

Reply:
What do you think about only pointing to the paragraph above "Overriding default click action"? There's a good example of how to set `ffPreventDefault`

Reply:
Yes, that sounds reasonable as long as it is clear what `ffPreventDefault` has to be set to.

Reply:
so i added the info about the expected value of `ffPreventDefault` and point to section "Overriding default click action" for more info and example usage